your_daily_dose_malware/backends/malshare.py

```diff
@@ -0,0 +1,46 @@
+import os
+
+import requests
+import click
+from dotenv import load_dotenv
+from rich.progress import Progress
+
+load_dotenv()
+
+
+def scrap_malshare():
+    with Progress() as progress:
+        api_key = os.getenv("MALSHARE_API_KEY")
+        print(api_key)
+        try:
+            response = requests.post(
+                "https://malshare.com/api.php",
+                verify=True,
+                params={"api_key": api_key, "action": "getlist"},
+            )
+            response.raise_for_status()
+            hashes = response.json()
+        except Exception as err:
+            raise err
+        try:
+            print(f"  - Malshare: {len(hashes)} Samples")
```

Comment:
Shall we segregate methods as verification and downloading?

Reply:
Maye you can expand on your idea. I dont see whay you mean.

Reply:
then what should we do now ??